cli_tools/gce_vm_image_export/exporter/exporter.go

```diff
@@ -46,6 +47,10 @@ const (
 	SourceImageFlagKey    = "source_image"
 )
 
+const (
+	logPrefix = "[image-export]"
```

Comment:
Prefix should end with a space. Otherwise, log message will be concatenated directly with the "]".

Applies for other prefixes.

Reply:
I added a space when SetPrefix. For gcs logging, the space is added when writing the actual log, so I won't add the space for both prefix.